### PR TITLE
Translations of "Status by goal"

### LINKED ISSUE #68: Translation of "Status by goal"
The phrase "Status by goal" is needed for Open SDG. See https://github.com/open-sdg/open-sdg/issues/113.

### PR DESCRIPTION
**Pull request by brockfanning**
Here is a draft of translations of "Status by goal". These were done with Google Translate, so careful review is needed.

Confirmed so far:
- [x] Arabic
- [ ] Armenian
- [x] Chinese
- [ ] French
- [x] German
- [ ] Russian
- [x] Spanish

Fixes #68

### CODE REVIEW

**Comment by JustusCas** [comment]
The German Translation is correct

**Comment by brockfanning** [comment]
Thanks @JustusCas. I'm reaching out to some translators about the other languages.

**Comment by brockfanning** [comment]
Got confirmation of the Arabic translation.

File: translations/es/status.yml [hunk 1]
```diff
@@ -3,3 +3,4 @@ overall_reporting_status: Estado general de informes
 reported_online: Reportado en línea
 reporting_status: Estado de informes
 statistics_in_progress: Estadísticas en progreso
+status_by_goal: Estado por objetivo
```
Comment (dougmet) [inline translations/es/status.yml]: A (non-native) spanish speaker said it should be "estados por objetivos" (note plural in estados). But I'd want verification.

**Comment by brockfanning** [comment]
I feel like the intention here is more of a singular type of thing; a more verbose statement might be "a status for each goal". But I can't speak Spanish so I'm no authority.

File: translations/zh-Hans/status.yml [hunk 2]
```diff
@@ -3,3 +3,4 @@ overall_reporting_status: 整体上报状况
 reported_online: 已在线上上报
 reporting_status: 上报状况
 statistics_in_progress: 统计在进行中
+status_by_goal: 目标状态
```
Comment (holypeony) [inline translations/zh-Hans/status.yml]: It should be 目标状况 instead of 目标状态.

**Comment by brockfanning** [comment]
I'll go ahead and merge this - I think the remaining will need to be audited later.

**Pull request merged**